mmv1/third_party/terraform/tests/resource_eventarc_channel_test.go.erb

```diff
@@ -0,0 +1,175 @@
+<% autogen_exception -%>
+package google
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	dcl "github.com/GoogleCloudPlatform/declarative-resource-client-library/dcl"
+	<% if version == "ga" -%>
+	eventarc "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/eventarc"
+	<% else -%>
+	eventarc "github.com/GoogleCloudPlatform/declarative-resource-client-library/services/google/eventarc/beta"
+	<% end %>
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccEventarcChannel_BasicHandWritten(t *testing.T) {
```

Comment:
Given we're handwriting the tests here, we'll want a slightly different set of coverage than the generated ones would have provided. Per https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/README.md#testing we'll want a simple test and an update test, rather than one hybrid one. I'd suggest:

* One `_basic` test provisioning the resource with a minimal number of fields set with values
* A second called `_cryptoKeyUpdate` with two stages setting and updating the `crypto_key_name` value.

Reply:
Acknowledged

Reply:
Done